pkgs/top-level/perl-packages.nix

```
@@ -18419,8 +18419,8 @@ let
   SysVirt = buildPerlModule rec {
     pname = "Sys-Virt";
     version = "6.3.0";
-    src = assert version == pkgs.libvirt.version; fetchgit {
-      url = "git://libvirt.org/libvirt-perl.git";
+    src = fetchgit {
+      url = "https://gitlab.com/libvirt/libvirt-perl.git";
       rev = "v${version}";
       sha256 = "0m75g0kf2dmllx5c1wxzszm1zakiqjighcsjbgq66674pvqf5mk7";
     };
```

Sorry, looking at it a bit closer, 6.3.0 is available on CPAN so it would probably be better to get the source from there.

```suggestion
   src = fetchurl {
      url = "mirror://cpan/authors/id/D/DA/DANBERR/Sys-Virt-v6.3.0.tar.gz";
      sha256 = "6333fe3c554322fec5a3e1890b08a4ea4f39b0fbb506b3592688a5785a488f39";
    };
```

---

Agreed.